src/core.js

```diff
@@ -10,23 +10,21 @@ var CryptoJS = CryptoJS || (function (Math, undefined) {
      *
      * As Math.random() is cryptographically not safe to use
      */
-    var secureRandom = function () {
+    var cryptoSecureRandomInt = function () {
         // Native crypto module on NodeJS environment
         try {
-            // Crypto from global object
-            var crypto = global.crypto;
+            // Native rypto from global object or import via require
+            var crypto = global.crypto || require('crypto');
 
-            // Create a random float number between 0 and 1
-            return Number('0.' + crypto.randomBytes(3).readUIntBE(0, 3));
+            return crypto.randomBytes(4).readInt32LE();
         } catch (err) {}

```

Comment:
Please check gracefully for the environment. 
Besides that I'm unaware of why crypto should be available in global in NodeJs. Only use require if require exists.
Exception management is expensive.

Reply:
crypto is available on global in NodeJS, I don't know why bit it just is.

But I agree to add checks.